webpack.config.test.js

```diff
@@ -0,0 +1,8 @@
+const nodeExternals = require( 'webpack-node-externals' );
+
+const baseConfig = require( './webpack.config' );
+
+const config = module.exports = Object.assign( {}, baseConfig, {
+	target: 'node',
+	externals: [ nodeExternals() ],
```

Comment:
I see it in `mocha-webpack`'s docs, and even upon reading the docs of `webpack-node-externals` itself, it's still not clear why this is necessary. Does it make the build more performant? Is it any different than:

```js
resolve: {
	modules: baseConfig.resolve.modules.filter( ( source ) => source !== 'node_modules' )
}
```